includes/api/ApiQueryUserContributions.php

```diff
@@ -156,26 +149,42 @@ private function prepareQuery() {
 			$encUser = $this->getDB()->strencode( $continue[0] );
 			$encTS = wfTimestamp( TS_MW, $continue[1] );
 			$op = ( $this->params['dir'] == 'older' ? '<' : '>' );
-			$this->addWhere(
-				"rev_user_text $op '$encUser' OR " .
-				"(rev_user_text = '$encUser' AND " .
-				"rev_timestamp $op= '$encTS')"
+			// SUS-807
```

Comment:
The block below got pretty long and it does not give a clue, what the query will be like.  Please, paste an example SQL query in a comment, formatted in a readable way.  It will help.